src/main/java/com/endava/web/application/infrastructure/restcontroller/conversion/EmployeeDtoToEmployee.java

```diff
@@ -0,0 +1,21 @@
+package com.endava.web.application.infrastructure.restcontroller.conversion;
+
+import com.endava.web.application.domain.model.Employee;
+import com.endava.web.application.infrastructure.restcontroller.dto.EmployeeDto;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmployeeDtoToEmployee implements Converter<EmployeeDto, com.endava.web.application.domain.model.Employee> {
```

Comment:
why do you import fully qualified path ?

Reply:
After a correction it was generated by Intellij and I did not saw it. Deleted. Done!